pkg/cmd/provisioning/ibmcloud/create_shared_secrets.go

```diff
@@ -29,8 +29,8 @@ type: Opaque`
 	manifestsDirName = "manifests"
 )
 
-// APIKeyEnvVar is the environment variable name containing an IBM Cloud API key
-const APIKeyEnvVar = "IC_API_KEY"
+// APIKeyEnvVars is the environment variable names containing an IBM Cloud API key
```

Comment:
// APIKeyEnvVars is a list of environment variable names containing an IBM Cloud API key

Reply:
fixed it.